sd_jwt/bin/sd_jwt

```diff
@@ -191,10 +202,10 @@ for hash in sdjwt_at_holder._hash_to_decoded_disclosure:
     encoded_json = sdjwt_at_holder._base64url_decode(b64).decode("utf-8")
 
     iid_payload += f"__Disclosure for `{claim_name}`:__\n"
```

Comment:
I think maybe one more newline would help the final output, which looks kinda funky now (https://drafts.oauth.net/oauth-selective-disclosure-jwt/danielfett/recurse-sd-jwt/draft-ietf-oauth-selective-disclosure-jwt.html#section-5.2-28).

```suggestion
    iid_payload += f"__Disclosure for `{claim_name}`:__\n\n"
```

Or put the line that was moved back down where it was.

Reply:
and update the examples again...

Reply:
I think it'll be good to go after that.